tests/solana_tests/tags.rs

```diff
@@ -184,10 +184,10 @@ fn functions() {
         }
 
         contract b {}"#,
-        Target::default_substrate(),
+        Target::Solana,
     );
 
-    assert_eq!(ns.diagnostics.len(), 3);
+    assert_eq!(ns.diagnostics.len(), 5);
```

Comment:
Why has this changed?

Reply:
Substrate does not like `contract b {}` without any functions and produces an error `contracts without public storage or functions are not allowed on Polkadot. Consider declaring this contract abstract: 'abstract contract b'`. The error prevents unused variable from running.

The test is looking for tags.